articles/ux-markup/properties.md

```diff
@@ -38,23 +42,29 @@ Property bindings can be *two-way* (default), *read-only* (`{ReadProperty ...}`)
 
 In the following example, the control defines a new property called `BackgroundColor`, with a default value (set on the root node) of `#f00`. This color is then property-bound to the `Color` property of a background rectangle with rounded corners. 
 
+```xml
 	<Panel ux:Class="MyButton" BackgroundColor="#f00">
 		<float4 ux:Property="BackgroundColor" />
 		<Text>SUBMIT</Text>
 		<Rectangle Layer="Background" Color="{Property BackgroundColor}" CornerRadius="10" />
 	</Panel>
+```
 
 When the component is instantiated, the user can either leave the default `BackgroundColor`, or set a new one.
 
+```xml
 	<MyButton />  <!-- this one will keep the default color -->
 	<MyButton BackgroundColor="#0f0" /> <!-- this one will use green instead -->
+```
```

Comment:
Hmm. this reminds me... can't we just use comments instead of those arrows? We're free to reformat the code a bit to make it a bit cleaner..

Reply:
Normal code comments? I don't think it will work with xml

Reply:
I meant xml comments. But it's not important for this pr, just a thought.